bluebutton-data-model-rif/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/RifDataloadHistory.java

```diff
@@ -0,0 +1,167 @@
+package gov.hhs.cms.bluebutton.data.model.rif;
+
+/**
+ * Entity that stores when each RIF dataload file is loaded to the PostgreSQL database
+ */
```

Comment:
Can you move this comment down so it's attached to the class? (Above the current line 17.)

Reply:
Done